### PR TITLE
pkp/pkp-lib#2559 custom autoloaders ##kaschioudi/2559-fix##

### PR DESCRIPTION
**Pull request by kaschioudi**
@asmecher

### CODE REVIEW

**Comment by kaschioudi** [comment]
@asmecher : looks like the failure is not related to my change. Am I correct?

**Comment by asmecher** [comment]
It looks like the FreeCite API is not responding as anticipated. I'm not sure if it's a temporary thing or not, but I agree, it doesn't look related to your changes.

File: classes/core/Application.inc.php [hunk 1]
```diff
@@ -32,6 +32,19 @@
...
+		parent::__construct();
+
+		// Register custom autoloader function for OJS namespace
+		spl_autoload_register(function($class) {
+			$prefix = 'OJS\\';
+			$rootPath = BASE_SYS_DIR . "/classes";
+			customAutoload($rootPath, $prefix, $class);
+		});
+	}
+	/**
```
Comment (asmecher) [inline classes/core/Application.inc.php]: Needs a space between blocks.

Comment (asmecher) [inline classes/core/Application.inc.php]: Still need a space here.

Comment (kaschioudi) [inline classes/core/Application.inc.php]: Done.

**Comment by asmecher** [comment]
I believe this was replaced by https://github.com/pkp/ojs/commit/b7389f1529b92de7380899c753c859c8c824aa33.

**Pull request closed without merging**